README.md

```diff
@@ -21,7 +21,6 @@ See the README in each directory for more information.
 
 * Install Lean, following the instructions [here](https://leanprover.github.io/lean4/doc/setup.html).
 * `cd cedar-lean`
-* `source ../cedar-drt/set_env_vars.sh`
```

Comment:
This step is needed to successfully `lake build` on AL2, at least prior to this PR. Did we check that these instructions work on AL2?

Reply:
Oof. Forgot about the `LD_PRELOAD` trick in that script. Will add this step back.